src/styles/index.css

```diff
@@ -1,6 +1,8 @@
+@import "./variables";
+
 /* Variables */
 :root {
-  --rent-strike-color: #DB3A1D;
+  --rent-strike-color: #db3a1d;
```

Comment:
Should we stick with CSS variables? [They're not supported by IE11](https://caniuse.com/#search=css%20var) so I thought it would be good to use an alternative like PreCSS or Sass.

Reply:
Using Sass via `node-sass` as it's a bit easier to lint and auto format.